composer.json

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "aspirepress/aspire-update",
+    "description": "Update plugins and themes for WordPress.",
+    "type": "wordpress-plugin",
+    "license": "GPL-3.0-or-later",
+    "authors": [
+        {
+            "name": "AspirePress"
+        }
+    ],
+    "require": {
+        "php": ">=7.4.0"
+    },
+    "require-dev": {
+        "yoast/phpunit-polyfills": "^1.1.0"
```

Comment:
Don't we also need phpunit as a dev dependency?

Reply:
`yoast/phpunit-polyfills` [has our back](https://github.com/Yoast/PHPUnit-Polyfills/blob/3.x/composer.json).